### ISSUE TITLE
Unsigned miltisig transaction are show as signed (can't sign raw transaction)

### ISSUE THREAD

**Issue by kot-begemot** [pooler/electrum-ltc#209]
**Wallet version**: 3.3.4.1
**TL;DR**: I am generating multisig transaction remotely and then sign it using using ElecturmLTC wallet.

**Description**
I am building an MVP and testing certain workflow. for test purposes, I've set up 2 separate multisig 2-of-2 electum ltc wallet. My set up is as follows

    wallet 1: has master seed of account 1 and xpub of account 2
    wallet 2: has master seed of account 2 and xpub of account 1

Now, I am creating transaction remotely, with MVP service I am testing, and loading this transaction to wallet 1 to sing

```
0100000001f2300fa00ba650732bea1516c5b09446807d50c3d341d1460c8a2fcba5f2553b0000000000ffffffff01780f0f00000000001976a91454b38572e8c3c951c0aa415a5431825f9d06099f88ac00000000
```

Here is a screen capture
![vokoscreen-2019-03-04_14-59-38](https://user-images.githubusercontent.com/780077/53735118-a40b6880-3e8e-11e9-9d9d-494c48856857.gif)
 
I wonder why this happens? If there is anything I can do to force electrum-ltc to know that transaction is unsinged by both parties?

I tried to generate transaction containing only redeem script (prefixed wih OP_0 without  signatures), bit that didn't work either

```
0100000001f2300fa00ba650732bea1516c5b09446807d50c3d341d1460c8a2fcba5f2553b000000004900475221028c895128a8113bb5d599b4f463df685220ba726fc664013b3df56cb0bae74dc42102c09fdcf2cd2281ac5d35dedb8b19499db847fad469c151428d28c9cf1aabf43552aeffffffff01780f0f00000000001976a91454b38572e8c3c951c0aa415a5431825f9d06099f88ac00000000
```

I investigated underlying logic and it seems to me that transaction is not recognized as p2sh at all. Here is what I did

```
~/experiments/electon-ltc/$ python3
>>> from electrum_ltc import transaction
>>> raw = "0100000001f2300fa00ba650732bea1516c5b09446807d50c3d341d1460c8a2fcba5f2553b000000004900475221028c895128a8113bb5d599b4f463df685220ba726fc664013b3df56cb0bae74dc42102c09fdcf2cd2281ac5d35dedb8b19499db847fad469c151428d28c9cf1aabf43552aeffffffff01780f0f00000000001976a91454b38572e8c3c951c0aa415a5431825f9d06099f88ac00000000"
>>> tx = transaction.Transaction(raw)
>>> tx.deserialize()
{'partial': False, 'version': 1, 'segwit_ser': False, 'inputs': [{'prevout_hash': '3b55f2a5cb2f8a0c46d141d3c3507d804694b0c51615ea2b7350a60ba00f30f2', 'prevout_n': 0, 'scriptSig': '00475221028c895128a8113bb5d599b4f463df685220ba726fc664013b3df56cb0bae74dc42102c09fdcf2cd2281ac5d35dedb8b19499db847fad469c151428d28c9cf1aabf43552ae', 'sequence': 4294967295, 'type': 'unknown', 'address': None, 'num_sig': 0}], 'outputs': [{'value': 987000, 'type': 0, 'address': 'LSwp5geEBjqRf45Dmmq2gbf2jtRBByEyj3', 'scriptPubKey': '76a91454b38572e8c3c951c0aa415a5431825f9d06099f88ac', 'prevout_n': 0}], 'lockTime': 0}
>>> tx.is_complete()
True
>>> tx.signature_count()
(0, 0)
>>> tx.inputs()
[{'prevout_hash': '3b55f2a5cb2f8a0c46d141d3c3507d804694b0c51615ea2b7350a60ba00f30f2', 'prevout_n': 0, 'scriptSig': '00475221028c895128a8113bb5d599b4f463df685220ba726fc664013b3df56cb0bae74dc42102c09fdcf2cd2281ac5d35dedb8b19499db847fad469c151428d28c9cf1aabf43552ae', 'sequence': 4294967295, 'type': 'unknown', 'address': None, 'num_sig': 0}]
```

So, my question is, what shall i do, in order to make sure that my unsigned multisig transaction is recognized as unsigned?

P.S. Same procedure worked with Electrum for BTC. Somehow I was able to sign BTC transaction with empty scriptSig field with both wallets.

**Comment by kot-begemot** [comment]
Seems like I can't sign transaction now with Electrum either. I will ask same question there. If they will provide answer earlier, I'll close this issue.
